analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -729,11 +729,15 @@ def start_workers(actions_map, actions, analyzer_config_map,
     def signal_handler(signum, frame):
         try:
             pool.terminate()
+            pool.join()
```

Comment:
Was this `join()` operation necessary for some reason?

Reply:
Its explained in the summary.